### PR TITLE
Fix bugs on BP page

### PR DESCRIPTION
**Pull request by AngeloCG97**
## Handle large BP names

#591
#557

### What does this PR have?
1. Fix show button vote
1. Fix broken large titles on card component

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (xavier506) [inline package.json:1]: why do we need this package.json file?

File: .husky/pre-commit [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm test
```
Comment (xavier506) [inline .husky/pre-commit:4]: npm ?

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]
@AngeloCG97 I detect two errors related with what you did,

Error n1: Rating don't appear or well is being cropped by small sizes
image

Error n2: Card are overlaping
image


Let me know if you can fix it in this PR or if you prefer to create new cards for that issues

**Comment by AngeloCG97** [comment]
I work this bugs on other PR.

**Review by leisterfrancisco — APPROVED** [review]

**Pull request merged**